SETUP.md

```diff
@@ -20,21 +20,21 @@ For a minimal setup, you only need to install [SBT (the Scala Build Tool)](http:
 
     Choose whatever version is being [used in continuous integration](.github/workflows/install-circt/action.yml)
     ```bash
-    wget -q -O - https://github.com/llvm/circt/releases/download/firtool-1.38.0/firrtl-bin-ubuntu-20.04.tar.gz | tar -zx
+    wget -q -O - https://github.com/llvm/circt/releases/download/firtool-1.58.0/firrtl-bin-linux-x64.tar.gz | tar -zx
```

Comment:
Can this instead point at latest? The way CD of CIRCT into Chisel is setup, this should be safe. This needs some changes to the scripts here to avoid referencing a version.

Reply:
Tried it with the latest for download with wget, but it did not work. However, putting following string into a browser works: https://github.com/llvm/circt/releases/latest

Reply:
Try: https://github.com/llvm/circt/releases/latest/download/circt-full-shared-linux-x64.tar.gz

For some reason the order flips when using the latest. "latest/download" instead of "download/<release>".

Alternatively, this file would need to be generated and put on the website pulling information out of `etc/circt.json`. I'm fine with just the generic directions that talk about the latest version and we don't have to keep them up to date.